### PR TITLE
[Merged by Bors] - Getting start docs

### PR DESCRIPTION
**Pull request by adwk67**
# Description

Addition of "getting started" documentation

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/getting_started/pages/installation.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+There are 2 ways to run Stackable Operators
+
+1. Using xref:stackablectl::index.adoc[]
+
+1. Using Helm
+
+=== stackablectl
+
+stackablectl is the command line tool to interact with Stackable operators and our recommended way to install Operators.
```
Comment (razvan) [inline docs/modules/getting_started/pages/installation.adoc:24]:
```suggestion
`stackablectl` is the command line tool to interact with Stackable operators and our recommended way to install Operators.
```

File: docs/modules/getting_started/pages/installation.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Follow the xref:stackablectl::installation.adoc[installation steps] for your platform.
+
+After you have installed stackablectl run the following command to install the Spark-k8s Operator:
```
Comment (razvan) [inline docs/modules/getting_started/pages/installation.adoc:27]:
```suggestion
After you have installed `stackablectl` run the following command to install the Spark-k8s operator:
```

File: docs/modules/getting_started/pages/installation.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+[INFO ] Installing commons operator
+[INFO ] Installing secret operator
+[INFO ] Installing spark-k8s operator
+----
+
+TIP: Consult the xref:stackablectl::quickstart.adoc[] to learn more about how to use stackablectl. For example, you can use the `-k` flag to create a Kubernetes cluster with link:https://kind.sigs.k8s.io/[kind].
+
+=== Helm
+
+You can also use Helm to install the Operators. Add the Stackable Helm repository:
```
Comment (razvan) [inline docs/modules/getting_started/pages/installation.adoc:46]:
```suggestion
You can also use Helm to install the operator. Add the Stackable Helm repository:
```

File: docs/modules/getting_started/pages/installation.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+include::example$code/getting_started.sh[tag=helm-add-repo]
+----
+
+Then install the Stackable Operators:
+[source,bash]
+----
+include::example$code/getting_started.sh[tag=helm-install-operators]
+----
+
+Helm will deploy the Operators in a Kubernetes Deployment and apply the CRDs for the SparkApplication (as well as the CRDs for the required operators). You are now ready to create a Spark job in Kubernetes.
```
Comment (razvan) [inline docs/modules/getting_started/pages/installation.adoc:58]:
```suggestion
Helm will deploy the operators in a Kubernetes Deployment and apply the CRDs for the `SparkApplication` (as well as the CRDs for the required operators). You are now ready to create a Spark job.
```

**Review by razvan — COMMENTED** [review]
Some suggestions and comments.

File: docs/modules/getting_started/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
+= Getting started
+
+This guide will get you started with Airflow using the Stackable Operator. It will guide you through the installation of the Operator and its dependencies, setting up your first Airflow cluster and connecting to it, and viewing and running one of the example workflows (called DAGs = Direct Acyclic Graphs).
```
[the remark this answers is not in the document]
Comment (adwk67) [reply docs/modules/getting_started/pages/index.adoc]: Yes, corrected now.

File: docs/modules/getting_started/pages/installation.adoc [hunk 6]
```diff
@@ -0,0 +1,62 @@
+= Installation
+
+On this page you will install the Stackable Spark-on-Kubernetes operator as well as the Commons and Secret operators which are required by all Stackable operators.
+
+== Dependencies
+
+The Spark-k8s Operator does not install a cluster or a set of nodes, but rather a SparkApplication that creates a spark-submit job, delegating this to the Spark driver, which in turn spins up executors for the duration of the job. Spark job dependencies can thus vary in their scope and detail. In this guide we are going to keep things simple and look at executing a Spark job that has a minimum of dependencies.
```
Comment (razvan) [inline docs/modules/getting_started/pages/installation.adoc:7]: Well then, let's write it like that :) Here is a suggestion:

```suggestion
Spark applications almost always require dependencies like database drivers, REST api clients and many others. These dependencies must be available on the `classpath` of executors and sometimes drivers. There are multiple ways to provision Spark jobs with such dependencies, some are built into Spark it's self while others are implemented at the operator level. In this guide we are going to keep things simple and look at executing a Spark job that has a minimum of dependencies.
```

**Review by razvan — APPROVED** [review]
Approved but with one more suggestion.

**Review by razvan — COMMENTED** [review]
It just occurred to me that there is no changelog entry for this pr. I would add one.

File: docs/modules/getting_started/pages/installation.adoc (hunk 6 above)
Comment (adwk67) [reply docs/modules/getting_started/pages/installation.adoc:7]: Thanks - much better! (just made a couple of minor tweaks to this)

**Comment by adwk67** [comment]
> It just occurred to me that there is no changelog entry for this pr. I would add one.

Added.

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**